scripts/generate-bindata.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"crypto/sha1"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/shurcooL/vfsgen"
+)
+
+func needsUpdate(dir string, filename string) (bool, []byte) {
+	needRegen := false
+	_, err := os.Stat(filename)
+	if err != nil {
+		needRegen = true
+	}
+
+	oldHash, err := ioutil.ReadFile(filename + ".hash")
+	if err != nil {
+		oldHash = []byte{}
+	}
+
+	hasher := sha1.New()
+
+	err = filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
```

Comment:
So, we're basically doing make's job. 😆

Reply:
Yup. That's what you get when you mix two competing build systems ...